src/formio/templates/field.ejs

```diff
@@ -1,19 +1,55 @@
-<div class="utrecht-form-field utrecht-form-field--openforms">
-{% if (!ctx.label.hidden && ctx.label.labelPosition !== 'bottom') { %}
-  {{ ctx.labelMarkup }}
-{% } %}
+{% if (ctx.component.type === 'radio' || ctx.component.type === 'selectboxes') { %}
+  <div class="utrecht-form-fieldset">
+    <fieldset
+      class="utrecht-form-fieldset__fieldset utrecht-form-fieldset--html-fieldset"
+      {% if (ctx.component.type === 'radio') { %}role="radiogroup"{% } %}
+      {% if (ctx.component.description) { %}aria-describedby="{{ctx.id}}-description"{% } %}
+    >
 
-{% if (ctx.label.hidden && ctx.label.className && ctx.component.validate.required) { %}
-  <label class="utrecht-form-label utrecht-form-label--openforms-required {{ctx.label.className}}"></label>
-{% } %}
+      <legend class="utrecht-form-fieldset__legend utrecht-form-fieldset__legend--html-legend utrecht-form-field__label">
+        <span class="utrecht-form-label utrecht-form-label--openforms-required">
+          {{ ctx.t(ctx.component.label, { _userInput: true }) }}
+          {% if (!ctx.component.validate.required && !ctx.requiredFieldsWithAsterisk) { %}
+            {{ ctx.t('(optional)') }}
+          {% } %}
+          {% if (ctx.component.tooltip) { %}
+            <i ref="tooltip" class="{{ctx.iconClass('question-sign')}}"></i>
+          {% } %}
+        </span>
+      </legend>
 
-{{ctx.element}}
+      {{ctx.element}}
 
-{% if (!ctx.label.hidden && ctx.label.labelPosition === 'bottom') { %}
-  {{ ctx.labelMarkup }}
-{% } %}
+      {% if (ctx.component.description) { %}
+        <div
+          class="utrecht-form-field-description utrecht-form-field-description--openforms-helptext"
+          id="{{ctx.id}}-description"
+        >
+          {{ctx.t(ctx.component.description)}}
```

Comment:
`ctx.component.description` is already translated server side to handle form variable interpolation. This probably ends up being a no-op. But there's a minute chance double translations may happen. Like the translation chain: streepje => bar => café.

Reply:
good catch, I just copy pasted it around but let's patch this out.